docs/disclose.css

```diff
@@ -116,8 +116,8 @@ section.dark {
 
 section.dark svg {
   display: block;
-  margin-bottom: 1em;
   max-height: 4.5em;
+  margin: 1em auto;
```

Comment:
Let’s go with `margin: 0 auto 1em;` to avoid adding extra vertical space to the page.